utils.py

```diff
@@ -189,6 +189,35 @@ def get_predictions(output):
     indices = indices.view(bs,h,w) # bs x h x w
     return indices
 
+# This function is written by Ying @UIUC RSim group. In this case, we have assumed the batch size is 1.
+def getCoordinates(predict):
+    pupilPixel = np.where(predict[0].cpu().numpy() == 3)
+    xMin = min(pupilPixel[0])
+    xMax = max(pupilPixel[0])
+    yMin = min(pupilPixel[1])
+    yMax = max(pupilPixel[1])
+    delta = (xMax - xMin) - (yMax - yMin)
```

Comment:
I'm confused as to how this block works. Wouldn't this always be 0 or is pupilPixel a non-integer value?